packages/react-scripts/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 _Nothing yet_
 
+### Changed
```

Comment:
Matt thoughts on this. My thinking is that this is a significant change in output result so it should be breaking even though no change is required by the user. Erring on the side of caution